strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/repository/metadata/MavenMetadataGroupRepositoryComponent.java

```diff
@@ -162,11 +167,14 @@ public void updateMetadataInGroupsContainingRepository(final String storageId,
             return;
         }
 
-        updateMetadataInGroupsContainingRepository(repository, artifactRelativePath, artifactBasePathCalculation,
-                                                   mergeMetadata);
+        final List<Repository> updatedRepositoryGraphPathLeafs = new ArrayList<>();
+        updatedRepositoryGraphPathLeafs.add(repository);
+        updateMetadataInGroupsContainingRepository(repository, updatedRepositoryGraphPathLeafs, artifactRelativePath,
```

Comment:
Cosmetic: Same as my earlier comment -- when the parameter list spans across multiple lines, could you please put each parameter on a separate line? Thanks! :)